guides/user/kubernetes-v2/patch-manifest/index.md

```diff
@@ -0,0 +1,151 @@
+
+* __Name__
+
+  The name of your Kubernetes resource.
+
+
+## Specify your patch content
+
+The patch content is similar to a manifest in the Deploy (Manifest) stage. However, unlike the deploy manifest, this does not have to be the full resource manifest but only the portion you want to patch.
+
+Depending on your needs, there is more than one way to specify the patch content:
+
+* [Statically: directly in the pipeline](#specify-patch-content-statically)
+* [Dynamically: bound at runtime using an artifact](#specify-patch-content-dynamically)
+
+
+### Specify patch content statically
+
+You can enter the patch content YAML by hand. For instance, if you want to patch your manifest to add a new label, you will specify the following:
+
+```yaml
+metadata:
+  labels:
+    foo: bar
+```
+
+{%
+  include
+  figure
+  image_path="./in-pipeline.png"
+  caption="Notice that by selecting __Text__ as the __Manifest Source__, we get
+  to enter the manifest YAML by hand."
+%}
+
+### Specify patch content dynamically
+
+Like the [Deploy (Manifest) stage](/guides/user/kubernetes-v2/deploy-manifest/index.md#specify-manifests-dynamically), you can also reference an artifact as the source if you are storing your patch content externally. The artifact must be a text file containing the patch content.
+
+You can also setup the pipeline to trigger based on changes to the patch content:
+
+* [Consuming GitHub Artifacts](/guides/user/triggers/github)
+* [Consuming GCS Artifacts](/guides/user/triggers/gcs)
+
+Assuming you have declared an expected artifact upstream to your Patch
+manifest stage, you can reference it in the Patch configuration:
+
+{%
+  include
+  figure
+  image_path="./in-artifact.png"
+  caption="Notice that by selecting __Artifact__ as the __Manifest Source__, we
+  get to pick which upstream artifact to deploy."
+%}
+
+> __☞ Note__: Make sure that the __Artifact Account__ field matches an account
+> with permission to download the manifest.
+
+
+## Override artifacts
+
+The patch stage also allows you to [override artifacts](/guides/user/kubernetes-v2/deploy-manifest/index.md#override-artifacts) like in the deploy manifest stage.
+
+For instance, say you have a pipeline with a patch manifest stage with the following patch content:
+
+```yaml
+spec:
+  template:
+    spec:
+      containers:
+        - name: my-container
+          image: gcr.io/my-project/my-image
+```
+
+Now, if your pipeline was triggered due to a new Docker image tag being pushed to your Docker registry (say my-image:2.0), Spinnaker will override the version of the container image with the new version:
+
+```yaml
+#...rest of manifest
+containers:
+  - name: my-container
+    image: gcr.io/my-project/my-image:2.0
+```
+For more information on how this works, check out the [binding artifacts docs](/reference/artifacts/in-kubernetes-v2/#binding-artifacts-in-manifests).
+
+
+## Specify Patch Options
+
+You can also optionally specify the following options:
```

Comment:
I think you can remove one "option*". I recommend the first one, as "can" implies optional. 
"You can specify the following options:"